change/@azure-msal-node-892ff60a-4f9d-47e4-8217-04e90f1381e4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Fixing merge conflict",
```

Comment:
Please give this a more descriptive comment as this will be published in the changelog (and include the PR #)

Reply:
+1